pages/web_cloud/domains/transfer_incoming_generic_domain/guide.fr-fr.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Transférer son nom de domaine vers OVHcloud'
 excerpt: "Découvrez comment réaliser le transfert d'un nom de domaine vers OVHcloud"
-updated: 2024-01-22
+updated: 2024-03-03
```

Comment:
```suggestion
updated: 2024-01-22
```
As the changes are transparent for our customers (no new content), I'd be in favor of not updating the date.